metrics/models.go

```diff
@@ -203,19 +205,41 @@ type CompleteTestStatus struct {
 
 // TestRunStatus binds a TestRun to a CompleteTestStatus.
 type TestRunStatus struct {
-	Run    *base.TestRun
+	Run    *shared.TestRun
 	Status CompleteTestStatus
 }
 
+// TestRunsMetadata is a struct for metadata derived from a group of TestRun entities.
+type TestRunsMetadata struct {
+	// Deprecated. Store the IDs in TestRunIDs instead.
+	TestRuns   []shared.TestRun `json:"test_runs,omitempty" datastore:"TestRuns,omitempty"`
+	TestRunIDs []int64          `json:"-"`
+	StartTime  time.Time        `json:"start_time"`
+	EndTime    time.Time        `json:"end_time"`
+	DataURL    string           `json:"url"`
+}
+
+// LoadTestRuns fetches the TestRun entities for the PassRateMetadata's TestRunIDs.
+func (metadata *TestRunsMetadata) LoadTestRuns(ctx context.Context) (err error) {
```

Comment:
I feel like `shared` should have a `TestRunIDs` type with a helper function `GetTestRuns`, which would complement #231 . Providing such a bidirectional mapping is a better API, IMHO. Then this method can simply call `GetTestRuns`.

Reply:
Done over @ https://github.com/web-platform-tests/wpt.fyi/pull/235/files